roles/deploy_flask_app/README.md

```diff
@@ -0,0 +1,94 @@
+deploy_flask_app
+==================
+
+A role to deploy a simple flask application with RDS db cluster as backend.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* ec2:DescribeInstanceAttribute
+* ec2:DescribeInstanceStatus
+* ec2:DescribeInstances
+* ec2:RunInstances
+* ec2:CreateKeyPair
+* elb:CreateLoadBalancer
+* rds:DescribeDBInstances
+* rds:DescribeDBSubnetGroups
+* rds:ListTagsForResource
+
+Role Variables
+--------------
```

Comment:
When including the role in a playbook, `local_registry_port`, `sshkey_pair_name` and `workers_user_name` are required in addition to the vars listed below.